templating/namespaced_paths.rst

```diff
@@ -14,7 +14,7 @@ Take the following paths as an example:
 
 .. code-block:: twig
 
-    {% extends "AppBundle::layout.html.twig" %}
+    {% extends "AppBundle:layout.html.twig" %}
```

Comment:
This change looks wrong to me. When the midle part is empty (hence the double `::`) it means that the file is in the `Resources/views/` directory: `Resources/views/layout.html.twig`